plugins/module_utils/constants.py

```diff
@@ -31,3 +31,5 @@
 NODE_TYPE_MAPPING = {"tier_2": "tier-2-leaf", "remote": "remote-leaf-wan", "virtual": "virtual", "unspecified": "unspecified"}
 
 SPAN_DIRECTION_MAP = {"incoming": "in", "outgoing": "out", "both": "both"}
+
+MATCH_TYPE_MAPPING = {"all": "All", "at_least_one": "AtleastOne", "at_most_one": "AtmostOne", "none": "None"}
```

Comment:
fix sanity issues:

ERROR: Found 2 pep8 issue(s) which need to be resolved:
ERROR: plugins/module_utils/constants.py:35:110: W292: no newline at end of file
ERROR: plugins/modules/aci_vrf_leak_internal_subnet.py:262:1: E302: expected 2 blank lines, found 1